src/Magnum/EigenIntegration/Test/IntegrationTest.cpp

```diff
@@ -215,6 +231,78 @@ void IntegrationTest::matrixMatrix() {
     CORRADE_COMPARE_AS((cast<Eigen::Matrix<double, 2, 3>>(a)), b, EigenType);
 }
 
+
+void IntegrationTest::stridedArrayView() {
+    float data[4*5];
+    for(int i = 0; i < 4*5; ++i) data[i] = float(i);
+    Containers::StridedArrayView2D<float> view1(data, {4, 5});
+    auto mapped = arrayCast(view1);
+    auto view2 = arrayCast(mapped);
+    for(int i = 0; i < 4; ++i) {
+        for(int j = 0; j < 5; ++j) {
+            CORRADE_COMPARE(view1[i][j], view2[i][j]);
+        }
```

Comment:
```suggestion
        CORRADE_ITERATION(i);
        CORRADE_COMPARE_AS(view1[i], view2[i], TestSuite::Compare::Container);
```

You'll need `#include <Corrade/TestSuite/Compare/Container.h>` for this.